examples/package.json

```diff
@@ -1,6 +1,9 @@
 {
     "private": true,
     "dependencies": {
+        "@angular/core": "14",
+        "@angular/compiler": "14",
```

Comment:
Is this used?

Reply:
well... it's a peerDep of angular/compiler-cli so it complains, and I assume it would error since the compiler-cli would be missing somethings